doc/admin/deployment.md

```diff
@@ -856,12 +857,13 @@ DAOS Agent is a standalone application to be run on each compute node.
 It can be configured to use secure communications (default) or can be allowed
 to communicate with the control plane over unencrypted channels. The following
 example shows daos_agent being configured to operate in insecure mode due to
-incomplete integration of certificate support as of the 0.6 release.
+incomplete integration of certificate support as of the 0.6 release and
+configured to use a non-default agent configuration file.
```

Comment:
I would probably change this to say that certificate support is complete in 0.9.